.github/renovate.json5

```diff
@@ -42,6 +42,4 @@
   /*************************************************
    *** Repository-specific configuration options ***
    *************************************************/
-  // Don't leave dep. update. PRs "hanging", assign them to people.
-  "assignees": ["cevich"],
```

Comment:
Let's just update this to @edsantiago and myself I would say

Reply:
Oh wait no this is the default assignees, yeah I think we should drop them.
I was thinking about the VM image updates in https://github.com/containers/automation/blob/657247095b2961f0b8db013407397f939f954d51/renovate/defaults.json5#L223-L224

Reply:
Yeah they're a pain in the butt more than anything else.  For context, the only reason I set this (and probably others) to myself is b/c how I have my notifications setup: Participate or mention only.  In other words, w/o PR assignment I wouldn't ever see them.